common/placeholders.js

```diff
@@ -76,6 +84,15 @@ function table(value) {
   return tableFinal + `<tr><td><p>Total:</p></td><td><p>"${total}"</p></td></tr></table>`
 }
 
+function tableReceipt(placeholders) {
+  let head = ['Foodbank Name','Foodbank Address','Foodbank Website','Client Intake Number', 'Support Number','EIN Number']
+  let tableFinal = `<table>`
+  for (let i = 0; i < 6; i++) {
+    tableFinal += `<tr><td><p>"${head[i]}":</p></td><td><p>"${placeholders[i]}"</p></td></tr>`
```

Comment:
Think a better way to do this is something like:
```js
function renderReceipt(values) {
  const ids = ['organization', 'address', 'url', 'supportNumber', 'ein']
  let table = '<table>'

  for (let i = 0; i < ids.length; i++) {
    const placeholder = placeholders.find(p => p.id === ids[i])
    table += `<tr><td>${placeholder.label}: ${values[placeholder.id]}</td></tr>`
  }

  // then generate the items table
}
```
The `server/lib/mail/mail-helpers.js` `send` function already passes the settings fields so you shouldn't need to change the `bindPlaceholders` function in the mail generator.